io_scene_nif/modules/nif_import/property/shader/__init__.py

```diff
@@ -50,38 +50,42 @@ class BSShader(ABC):
     def __init__(self):
         self.b_mesh = None
 
+    # TODO [texture] Implement clamp on image wrapping
```

Comment:
We probably want to move this over to the node wrapper class?

Reply:
Yeah, makes sense if it has multiple uses.
According to - https://docs.blender.org/api/current/change_log.html#bpy-types-image it has been removed so hopefully we can implement an equivalent or bette, but was planning on looking at generally implementing node

Reply:
What we have is b_texture_node.extension, but that is not available for separate directions(X / Y).